tagr/storage/local.py

```diff
@@ -44,5 +47,11 @@ def dump_pickle(self, pickle_object, proj, exp, tag, filename):
         filename: filename to be exported
         """
         pickle.dump(pickle_object, open(
-            "{}-{}-{}-{}.pkl".format(proj, exp, tag, filename), "wb"
+            "{}/{}/{}/{}.pkl".format(proj, experiment, tag, filename), "wb"
         ))
+    
+    def build_path(self, proj, experiment, tag):
+        try:
+            os.makedirs("{}/{}/{}".format(proj, experiment, tag))
+        except OSError:
```

Comment:
Nice now people can't overwrite an experiment locally. So this works because `datetime` tags can never be duplicated. We should make the error message more explicit. Something like
"The directory waterflow-tagr/sunrise/testlocal already exists. If using the tag argument, please provide a new unique identifier"